### DISCUSSION TITLE
Ar doesn't work on android

### DISCUSSION THREAD

**Discussion by tom-deadline** [google/model-viewer#2280] [Q&A]
Hey guys, I have an issue with AR mode on android,
I've tried on several devices, but it doesn't work on any of them.
it keeps saying that the model couldn't load, it works just fine on iOS, but not on Android.
![Image from iOS](https://user-images.githubusercontent.com/44176198/114404080-6ec9bc80-9bb6-11eb-92f9-428a6942c634.jpg)

I've checked the model on https://vr.google.com/scene-viewer-preview and no errors or warnings
 I've also tried changing ar priorities 
 `ar-modes="webxr scene-viewer quick-look"`, 
 `ar-modes="scene-viewer webxr quick-look"`, on both cases the result is the same

what can I do to fix this?

**Comment by LaloYanez** [comment]
All I could think of without looking at the code is that the path for the gltf/glb is wrong. If thats the case you wouldnt be able to preview anything on the viewer but when entering AR on iOS you would get the model if the usdz path is correct.

**Reply by arthurrmp** [reply]
Don't know if that's the case but maybe it can help. 

I also can't see the models in AR if they are compressed by **[gltfpack](https://github.com/zeux/meshoptimizer)**

**Reply by tom-deadline** [reply]
@LaloYanez Than I wouldn't be able to view the model in 3d viewer either. And I can see the model in the 3D viewer. 
@arthurrmp And the model isn't compressed by gltfpack either.
on the official website, there is the same problem with the scene-viewer example.

**Comment by tom-deadline** [comment]
@elalish @arthurrmp 
Is there a way I can contact scene-viewer team? I know this place is for model-viewer disscussions, but can you guys guide me who to ask about this issue?

**Reply by elalish** [reply]
@tpsiaki @AdrianAtGoogle for the Scene Viewer issue. I believe a fix just went out, so try making sure everything is updated. It looks like WebXR is not working; are you serving on HTTPS? This is required for WebXR.
